pandas/tests/test_algos.py

```diff
@@ -648,7 +648,9 @@ def test_value_counts_uint64(self):
         expected = Series([1, 1], index=[-1, 2**63])
         result = algos.value_counts(arr)
 
-        tm.assert_series_equal(result, expected)
+        # 32-bit linux has a different ordering
```

Comment:
@gfyoung FYI it seems we have another skip on 32-bit in here as well, so not specific to uint64. I think there is a n issue somewhere. but pretty hard to debug.